templates/submission/status.html

```diff
@@ -74,10 +74,19 @@
         <div><a href="{{ url('problem_submit', submission.problem.code, submission.id) }}">{{ _('Resubmit') }}</a></div>
     {% endif %}
     {% if perms.judge.rejudge_submission and not submission.is_locked %}
+        {% compress js %}
```

Comment:
Why are we shoving this into a separate js file?

Reply:
Where should it go? The JS blocks in this file are in unrelated if blocks.

Reply:
Eh, seems fine, it'll get compressed.